--- trafficserver-6.2.0/mgmt/Cop.h.orig	2017-05-15 04:04:39.925375249 +0000
+++ trafficserver-6.2.0/mgmt/Cop.h	2017-05-15 04:04:03.597691938 +0000
@@ -22,6 +22,6 @@
 */
 
 static const int cop_sleep_time      = 10;     // 10 sec
-static const int cop_manager_timeout = 3 * 60; //  3 min
-static const int cop_server_timeout  = 3 * 60; //  3 min
+static const int cop_manager_timeout = 30 * 60; //  30 min
+static const int cop_server_timeout  = 30 * 60; //  30 min
 static const int cop_kill_timeout    = 1 * 60; //  1 min
